src/main/java/net/mcreator/generator/blockly/BlocklyBlockCodeGenerator.java

```diff
@@ -444,8 +444,8 @@ public void generateBlock(BlocklyToCode master, Element block) throws TemplateGe
 			dataModel.put("cbi", customBlockIndex);
 			dataModel.put("addTemplate", new ExtraTemplatesLinker(master));
 
-			AtomicReference<String> head = new AtomicReference<>("");
-			AtomicReference<String> tail = new AtomicReference<>("");
+			AtomicReference<String> head = new AtomicReference<>(master.getHeadSection());
```

Comment:
Please do this in a separate PR and also explain in this PR the need for this change